### PR TITLE
chore(deps): Bump Rust version to 1.81.0

### PR DESCRIPTION
**Pull request by bruceg**
Includes a load of automated changes and a couple surprises.

### CODE REVIEW

File: src/sinks/aws_s_s/sink.rs [hunk 1]
```diff
@@ -2,15 +2,6 @@ use super::{client::Client, request_builder::SSRequestBuilder, service::SSServic
 use crate::sinks::aws_s_s::retry::SSRetryLogic;
 use crate::sinks::prelude::*;
 
-#[derive(Clone, Copy, Debug, Default)]
-pub(crate) struct SqsSinkDefaultBatchSettings;
-
-impl SinkBatchSettings for SqsSinkDefaultBatchSettings {
```
Comment (jszwedko) [inline src/sinks/aws_s_s/sink.rs:8]: I'm wondering if we meant to use these 🤔

File: src/sources/aws_s3/sqs.rs [hunk 2]
```diff
@@ -361,15 +361,13 @@ impl IngestorProcess {
...
         let messages = self.receive_messages().await;
         let messages = messages
-            .map(|messages| {
+            .inspect(|messages| {
                 emit!(SqsMessageReceiveSucceeded {
                     count: messages.len(),
                 });
-                messages
             })
-            .map_err(|err| {
```
Comment (jszwedko) [inline src/sources/aws_s3/sqs.rs:370]: I'm happy to see `inspect_err` finally 🎉

**Review by jszwedko — APPROVED** [review]
Nice, thank you!

File: src/sinks/aws_s_s/sink.rs (hunk 1 above)
Comment (bruceg) [reply src/sinks/aws_s_s/sink.rs:8]: Maybe we _meant_ to, but it is completely unused now and so causing a complaint.

File: website/cue/reference/components/sources/base/statsd.cue [hunk 3]
```diff
@@ -64,7 +64,7 @@ base: components: sources: statsd: configuration: {
 			Whether or not to sanitize incoming statsd key names. When "true", keys are sanitized by:
 			- "/" is replaced with "-"
 			- All whitespace is replaced with "_"
-			- All non alphanumeric characters [^a-zA-Z_\\-0-9\\.] are removed.
+			- All non alphanumeric characters (A-Z, a-z, 0-9, _, or -) are removed.
```
Comment (buraizu) [inline website/cue/reference/components/sources/base/statsd.cue:67]:
```suggestion
			- All non-alphanumeric characters (A-Z, a-z, 0-9, _, or -) are removed.
```

**Review by buraizu — APPROVED** [review]
Approving with a minor suggestion

**Pull request merged**